here_for_change/municipalities/views.py

```diff
@@ -1,47 +1,74 @@
 from django.views.generic import ListView, DetailView
-from .models import Municipality, Ward
+from .models import Municipality, Ward, WardDetail as WD
```

Comment:
Let's not abbreviate unless we have to. WardDetail is fine.

Reply:
@cyber237

Reply:
@paulmwatson model and view name clash, had to abbreviate the model name to avoid this.

Reply:
@cyber237  I'd then use WardDetailModel or WardDetailView or similar.